pkg/policy/api/consumable_decision.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016-2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"fmt"
+)
+
+// ConsumableDecision is a decision when deciding ingress/egress consuming
+type ConsumableDecision byte
+
+const (
+	// UNDECIDED means unsed while the decision is still undecided
```

Comment:
Undecided means that the policy decision has not been made yet. I haven't introduced this code in this commit, I've just moved it into a separate package.